base-offline/layouts/partials/video_player.html

```diff
@@ -0,0 +1,14 @@
+{{ $archiveUrl := .archiveUrl }}
+{{ $archiveVideoFound := false }}
+{{ $staticResourcesBundle := site.GetPage "/static_resources" }}
+{{ $archiveVideos := $staticResourcesBundle.Resources.ByType "video" }}
+{{ range $archiveVideos }}
+  {{ if strings.Contains $archiveUrl .Name }}
+    {{ $archiveVideoFound = true }}
+  {{ end }}
+{{ end }}
```

Comment:
Good catch, I didn't see this in the docs. I did get this to work but had to modify it a little. The code as you suggested it does not work, because `.Match` is looking at the `.Name` property of each resource, so what you pass in needs to be what you are matching against `.Name`. Since the `.Name` property of resources is the filename, I had to parse the url and then the path:

```
{{ $archiveVideoFound := false }}
{{ $archiveUrl := .archiveUrl }}
{{ $parsedArchiveUrl := urls.Parse $archiveUrl }}
{{ $parsedArchiveUrlPath := path.Split $parsedArchiveUrl.Path }}
{{ $archiveFile := $parsedArchiveUrlPath.File }}
{{ $staticResourcesBundle := site.GetPage "/static_resources" }}
{{ $archiveVideos := $staticResourcesBundle.Resources.ByType "video" }}
{{ $archiveVideoSearchResuts := $staticResourcesBundle.Resources.Match $archiveFile }}
{{ $archiveVideoFound = gt (len $archiveVideoSearchResuts) 0 }}
```